fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/MessageValidator.kt

```diff
@@ -167,6 +179,17 @@ object MessageValidator {
       return this
     }
 
+    /**
+     * Helper method to check that a string is not empty.
+     *
+     * @param input the string to check
+     * @param field name of the field (to print in case of error)
+     */
+    fun isNotNegative(input: Int, field: String): MessageValidatorBuilder {
```

Comment:
The doc says that input is a string but it is defined as an Int. Is there a specific reason for that ?

Reply:
yeah my bad, thanks!